src/hotspot/share/opto/macroArrayCopy.cpp

```diff
@@ -1101,9 +1101,6 @@ void PhaseMacroExpand::expand_arraycopy_node(ArrayCopyNode *ac) {
     merge_mem = MergeMemNode::make(mem);
     transform_later(merge_mem);
 
-    RegionNode* slow_region = new RegionNode(1);
-    transform_later(slow_region);
-
```

Comment:
This cleanup does not seem related?

Reply:
Yes but it does not seem justified to create a separate PR for removing dead code in the same file.

Reply:
Okay then.